pandas/tests/groupby/test_value_counts.py

```diff
@@ -85,8 +85,10 @@ def rebuild_index(df):
     tm.assert_series_equal(left.sort_index(), right.sort_index())
 
 
-def test_series_groupby_value_counts_with_grouper():
+@pytest.mark.parametrize("categorical", [True, False])
```

Comment:
make another test as we want to use the observed fixture.

Reply:
pls also hard code the expected results for both cases (much easier to see / understand).

I would also expected this change to break something, aren't the current expected results wrong? or are we just not testing them.

Reply:
Thanks @jreback 

Current results aren't tested at the moment. The proposed changes to the test file without the corresponding changes to pandas/core/groupby/generic.py will result in a failed assertion. 

I am unsure about what your comment re observed fixture means, so just to clarify, I should :

1) roll back my changes to test_value_counts.py

2) leave existing function test_series_groupby_value_counts_with_grouper as it was

3) create new test for when the Series giving rise to the SeriesGroupBy is categorical and hardcode expected results

I'm not sure what "both cases" is referring to. Unless you're intending me to rewrite the existing function test_series_groupby_value_counts_with_grouper to use hardcoded expected results also?

Reply:
yes, simply use the observed fixed as an argument to groupby that's 2 cases to test. i wonder if these give the same result.

Reply:
I'm happy to write a test where the groupby has observed=True, however isn't it only relevant if the groupers are categorical, which is not the case here?

Reply:
hmm actually yeah observed is not relevant. ok pls make a new test in any event.